src/master-playlist-controller.js

```diff
@@ -820,14 +820,13 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
     if (!buffered.length) {
       // return true if the playhead reached the absolute end of the playlist
-      return absolutePlaylistEnd - currentTime <= Ranges.TIME_FUDGE_FACTOR;
+      return absolutePlaylistEnd - currentTime <= 0.1;
```

Comment:
I'm not super familiar with this... why not just change the value of TIME_FUDGE_FACTOR instead of making this a hard coded value?  Is TIME_FUDGE_FACTOR using in a bunch of spots?  

Might also be good to comment on why 0.1 seconds for future explorers.

Reply:
It is used in many different places. Increasing the `TIME_FUDGE_FACTOR` to `0.1` for everywhere might be a good idea, but it increases the scope of this change a lot and I'm not sure if the other places using this value require more precision.